src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDrive.ps1

```diff
@@ -0,0 +1,248 @@
+<#
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License. See License.txt in the project root for license information.
+#>
+
+<#
+.SYNOPSIS
+    Returns a list of all storage drives at a location.
```

Comment:
Does this Synopsis/Description make sense anymore?  Now you are getting storage drives for a given cluster.

Reply:
The description is from our team design doc, but your suggestion is better, I have updated synopsis & description.